### PR TITLE
[Dessert] - Mac mini post

### LINKED ISSUE #3: M1 Mac mini 使用半年体验 - Mac的新未来
# Outline

- 个人使用场景 + 用户画像
- 总体体验
	- 性能
	- 稳定性
	- 兼容性
	- 可扩展性
	- Apple设备间协作

- 对WWDC的展望
    - macOS
    - New Macs

# Milestone
- 6/1 - Outline complete
- 6/5 - Draft complete
- 6/6 - Review complete
- TBD - Release

### PR DESCRIPTION
**Pull request by Realvincentyuan**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes #3  (Mac mini post)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

# How Has This Been Tested?

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: docs/Soft_core_blog/hardware/2021-06-03-Mac-mini.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+
+## 2 使用体验
+### 2.1 兼容性 - 总体满意
+
+首先谈兼容性是因为M1芯片架构从x86架构转换到了ARM架构，很多朋友会担心是否之前Intel Mac上的软件无法使用，让新电脑中看不中用。为了解决软件兼容性的问题，Apple提出了两种解决方案：
+
+- [Rosetta 2](https://support.apple.com/en-us/HT211861)，会在第一次安装app时提示安装这个软件，随后其都会在需要的时候在后台运行，负责在安装为Intel Mac打造的app时将其转译成ARM架构Mac能运行的代码。
+<figure>
+  <img src="https://cdn.jsdelivr.net/gh/BulletTech2021/Pics/2021-6-5/1622900346241-image.png" width="600" />
+  <figcaption>按照Rosetta 2，图片来源：Apple</figcaption>
```
Comment (jz263) [inline docs/Soft_core_blog/hardware/2021-06-03-Mac-mini.md:27]: 安装？

**Review by jz263 — COMMENTED** [review]
除了一个wording问题其他都很棒

File: docs/Soft_core_blog/hardware/2021-06-03-Mac-mini.md (hunk 1 above)
Comment (Realvincentyuan) [reply docs/Soft_core_blog/hardware/2021-06-03-Mac-mini.md]: 多谢！已经修复，@jz263

**Comment by Realvincentyuan** [comment]
> 除了一个wording问题其他都很棒

Thanks Jeremy, plz approve my post :) @jz263

**Pull request merged**